src/Chameleon.php

```diff
@@ -92,6 +92,12 @@ public static function init() {
 			$GLOBALS[ 'egChameleonLayoutFile' ] = $GLOBALS[ 'wgStyleDirectory' ] . '/chameleon/' .
 				$GLOBALS[ 'egChameleonLayoutFile' ];
 		}
+
+		// set default skin theme
+		$themeFile = &$GLOBALS[ 'egChameleonThemeFile' ];
+		if ( DIRECTORY_SEPARATOR === '/' && !empty( $themeFile ) && $themeFile[0] !== '/' ) {
```

Comment:
That check should short circuit that logic on Windows. I added that to the existing layout check in this PR: https://github.com/ProfessionalWiki/chameleon/pull/250

There are 2 Windows related issues caused by that `/` check (#81, #249). From what I understand that constant should return `\` on Windows and bypass the logic. In general PHP should handle directory separators in a cross-platform way, but I believe in this case the comparison is happening on a simple string so then it doesn't find a `/` when a typical Windows path is provided (`C:\ABC\XYZ`).

Reply:
Ha, so much for never any complaints. 

If I understand it correctly, this mechanism is supposed to add the missing first part of the path to the default theme (resources/styles/themes/_light.scss), so with the additional check, does that still happen on Windows? And why?

Reply:
I took a closer look at this and my conclusion is that the whole `if` is unnecessary and the original `if` used for the layout path is probably also unnecessary. I copied that logic because I assumed there was some (perhaps legacy) requirement for it.

Firstly, the layout file config variable was updated to be a path (https://github.com/ProfessionalWiki/chameleon/pull/161/commits/e1b631623adfd00f0795fa1070cfd9cf3de46b82) - when declaring a config variable like that the default value will return a full path including the extension directory (docs: https://www.mediawiki.org/wiki/Manual:Extension_registration#Path). This means the _default_ path (for layout and theme) should be expanded correctly by MediaWiki (theoretically this should already be working on Windows as well).

Secondly, the docs suggest prepending all paths with `__DIR__` thus making them absolute in any case. This, at least on Linux, would not trigger the condition.

I still need to get my hands on a WAMP setup to prove this. However, the current Windows issues are caused by having  a `C:\...` style absolute path which then triggers that missing `/` condition.

The only cases currently handled by the `if` would be the following relative paths. However, this is not mentioned in the documentation.
```php
$egChameleonAvailableLayoutFiles = [
    "standard" => "skins/chameleon/layouts/standard.xml",
    "navhead"  => "skins/chameleon/layouts/navhead.xml",
];
$egChameleonLayoutFile = "layouts/stickyhead.xml";
$egChameleonThemeFile = "resources/styles/themes/_light.scss";
```

To summarise, without the `if`:
* default path defined in `skin.json` is already expanded by MW
* admins should always specify the full path (with `__DIR__`)
* it does not make sense to introduce the relative syntax support when there is only one theme in the extension

Therefore, I am going to remove the `if` in this PR.

Reply:
I think you are right! Full support to remove the if. 👍